src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.